README.md

```diff
@@ -1,52 +1,242 @@
 
-10. If circle CI checks are all green, you are all set!
+`npm install reshuffle-teamwork-connectors`
 
-// Keep documentation template below
+### Reshuffle Teamwork Connector
 
-## END
+This package contains a [Reshuffle](https://github.com/reshufflehq/reshuffle)
+connector to connect [Teamwork Online app APIs](https://github.com/moshie/teamwork-api).
 
-# reshuffle-_CONNECTOR_NAME_-connector
+The following example exposes an endpoint to return the data of a Task after an Update action.
 
-### Reshuffle _CONNECTOR_NAME_ Connector
+```js
+const { Reshuffle } = require('reshuffle')
+const { TeamworkConnector } = require('reshuffle-teamwork-connectors')
+
+const app = new Reshuffle()
+
+const connector = new TeamworkConnector(app,  {
+  apiKey: YOUR_API_KEY,
+  subdomain: YOUR_SUB_DOMAIN,
+  webhookPath: '/webhooks/teamwork'
+})
+
+connector.on({ type: 'TASK.UPDATED' }, async (event, app) => {
+  console.log('TASK.UPDATED')
+  console.log(event.eventCreator.id)
+  console.log(event.taskList.name)
+  console.log(event.task)
+})
+
+app.start()
+```
+
+### Table of Contents
+
+[Configuration Options](#configuration)
+
+#### Connector Events
+
+[Listening to Teamwork events](#listen)
+
+#### Connector Actions
+
+[SDK](#sdk) - Retrieve a full Teamwork sdk object
+
+
+### <a name="configuration"></a> Configuration options
+
+```js
+const app = new Reshuffle()
+const connector = new TeamworkConnector(app, {
+  apiKey: YOUR_API_KEY, 
+  subdomain: YOUR_SUB_DOMAIN, 
+  webhookPath: WEBHOOK_PATH 
+})
+```
+
+- apiKey - How to find the [API key](https://developer.teamwork.com/projects/apikey/key)
+- subdomain - https://[subdomain].teamwork.com/
+- webhookPath - The path component of the Webhook Endpoint URI for this project. The default value is '/webhooks/teamwork'.
+
+
+More details about the fields are described in [teamwork-api](https://github.com/moshie/teamwork-api)
+
+You can use the `webhookPath` to configure the url that Teamwork hits when it makes its calls to.
+For example - having the Reshuffle runtime URL `https://my-reshuffle.com` and `webhookPath='/webhook` will result in a complete webhook path of `https://my-reshuffle.com/webhook`.
+If you do not provide a `webhookPath`, Reshuffle will use the default webhook path for the connector which is `/webhooks/teamwork`.
+You will need to register this webhook with Teamwork. See [instructions](https://developer.teamwork.com/projects/webhooks/setup).
+
+
+### <a name="events"></a> Events
+
+#### <a name="listen"></a> Listening to Teamwork events
+
+To listen to events happening in Teamwork, you'll need to capture them with the connector's `on`
+function, providing a `TeamworkConnectorEventOptions` to it.
 
-This connector provides <description>.
 
-#### Configuration Options:
 ```typescript
-interface _CONNECTOR_NAME_ConnectorConfigOptions {
-  foo: string // foo description
-  bar?: number // bar description
+interface TeamworkConnectorEventOptions {
+  type: TWEventType // See bellow 
```

Comment:
Are we using shorthand (TW instead of Teamwork) with other connectors? I am not sure we should

Reply:
Now its TeamworkventType